VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/VitruvioModule.cpp

```diff
@@ -148,6 +148,26 @@ void SetInitialShapeGeometry(const InitialShapeBuilderUPtr& InitialShapeBuilder,
 	{
 		UE_LOG(LogUnrealPrt, Error, TEXT("InitialShapeBuilder setGeometry failed status = %hs"), prt::getStatusDescription(SetGeometryStatus))
 	}
+	
+	for (int32 UVSet = 0; UVSet < 8; ++UVSet)
+	{
+		std::vector<double> uvCoords;
+		for (const FInitialShapeFace& Face : InitialShape)
+        {
+            for (const auto& UV : Face.TextureCoordinates[UVSet].TextureCoordinates)
```

Comment:
make sure the TextureCoordinates are initialized, before accessing it, see Error, when opening default scene:
![image](https://user-images.githubusercontent.com/91744642/154222063-db39d295-273b-46ad-82b7-96923c5270bb.png)

Reply:
Good catch! Will fix that!